uncompressed katavasia of mid-pentecost draft

FrRaphaelDaly opened the pull request:
<!--
  Welcome! It looks like you are creating a pull request. We think that is great.
  In order to be able to enforce the CC BY-NC-ND 4.0 license most effectively,
  this form is pre-populated with a Contributor License Agreement (CLA),
  which is required if you want to contribute to this repository.
-->

### Submitter checklist

- [x] By creating this pull request, I represent that I have the right to waive copyright and related rights to my contribution, I agree that all copyright and related rights to my contributions are waived, and I acknowledge that Basil Crow is assigned the copyright to use and modify my contribution under the CC BY-NC-ND 4.0 license in perpetuity.

<!--
  Put an x between [ ] to assign copyright to Basil Crow.
-->

FrRaphaelDaly committed:
uncompressed katavasia of mid-pentecost draft

basil commented on unc_katavasia_mid-pentecost.byzx:
```diff
@@ -0,0 +1,3098 @@
+{
+  "version": "1.0",
+  "appVersion": "0.4.11",
```
You'll want to upgrade to the [latest release](https://github.com/neanes/neanes/releases/tag/v0.4.20) of the software, as there have been minor changes to the file format since version 0.4.11.

basil reviewed (commented):
Thanks for the PR! To get the ball rolling on this submission, let's start by integrating this score into this repository's standard editorial conventions. The file should be placed in the "Eortologion - Doxastarion/Pentecostarion" subdirectory and should be named something like "Katavasiae of Mid-Pentecost.byzx" for consistency with the existing directory structure and filenames. And the copyright assignment box in the PR template needs to be checked for all contributions to this repository.

The next step is updating the score to be consistent with [this repository's template](https://github.com/basil/byzantine-music/blob/master/Template.byzx). The easiest way to achieve this is to open the template, go to **File, Save As,** save the template with the new score's filename, and then copy/paste your existing score into the template with **Edit, Paste with Lyrics.** If this is too difficult, then let me know and I will push a commit that does this.

After that, the next step is adding drop caps to each katavasia (again, to be consistent with the other canons in this repository) and fixing up the lyrics. The lyrics are missing many punctuation marks and capital letters. I would recommend typing the entire text from the liturgical book in a text editor, checking it carefully against the liturgical book to make sure every capital letter and punctuation mark is identical, then pasting the result into https://juiciobrennan.com/hyphenator/ to choose the dictionary hyphenation points for each word. The result can be compared, line-by-line, with what is entered in the score, and any differences corrected. The final score should have identical capitalization and punctuation to the liturgical book, as well as dictionary hyphenation for each word.

That should keep you busy for a bit, and then when we come back we can start to take a look at the music and correct the various issues of orthography, as well as making the melody smoother and more rhythmically vibrant for a polished result.

FrRaphaelDaly committed:
Template and text

Template adopted
Capitalization and punctuation matched
Drop caps inserted
Corrections to music
